search_engine_parser/core/engines/google.py

```diff
@@ -87,7 +92,19 @@ def parse_single_result(self, single_result, return_type=ReturnType.FULL, **kwar
                 link_tag = desc_tag.find('a')
                 if link_tag:
                     raw_link = link_tag.get('href')
+                    results['hrefs'] = self.extract_href(raw_link)
                     results['links'] = self.parse_url(raw_link)
             desc = desc_tag.text
             results['descriptions'] = desc
         return results
+
+    def extract_href(self, url):
+        """Overwrite BaseSearch::extract_href."""
+        parsed = urlparse.urlparse(url)
+        url_qs = parse_qs(parsed.query)
+        if 'q' in url_qs:
+            return url_qs['q']
+        elif 'url' in url_qs:
```

Comment:
I haven't seen where google links return the url as a `url` param. Can you verify any instance this happens?

Reply:
Yes I found a few, though they were with `hl != 'en'`